awx/main/access.py

```diff
@@ -2750,6 +2750,9 @@ def can_add(self, data):
         else:
             return (self.check_related('workflow_approval_template', UnifiedJobTemplate, role_field='admin_role'))
 
+    def can_change(self, obj, data):
+        return self.user.can_access(WorkflowJobTemplate, 'change', obj.workflow_job_template, data=data)
```

Comment:
Instead of passing `data=data`, I would much prefer `data={}`. The data you get contain fields for modification of the workflow approval template, not fields for the WFJT, which is how it would be interpreted in this call.

I'm also happy to take your word that the approval node's WFJT cannot be modified and cannot be null, so that this is not complicated further.

Reply:
Agreed, `data={}` *is* probably what we want here.